app/src/main/java/com/yello/presentation/onboarding/activity/OnBoardingViewModel.kt

```diff
@@ -0,0 +1,175 @@
+package com.yello.presentation.onboarding.activity
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.map
+import com.example.domain.entity.MyCode
+import com.example.domain.entity.MyDepartment
+import com.example.domain.entity.MyFriend
+import com.example.domain.entity.MyGender
+import com.example.domain.entity.MyId
+import com.example.domain.entity.MyName
+import com.example.domain.entity.MySchool
+import com.example.domain.entity.MyStudentid
+
+class OnBoardingViewModel : ViewModel() {
+    val _school = MutableLiveData("")
+    val _department = MutableLiveData("")
+    val _studentid = MutableLiveData("")
+    val _name = MutableLiveData("")
+    val _id = MutableLiveData("")
+    val _gender = MutableLiveData("")
+    val _code = MutableLiveData("")
+
+    fun setSchool(school: String) {
+        _school.value = school
+    }
+
+    fun setDepartment(department: String) {
+        _department.value = department
+    }
+
+    fun setStudentid(studentid: String) {
+        _studentid.value = studentid
+    }
+
+    val _currentPage = MutableLiveData(0)
+    val currentPage: LiveData<Int> = _currentPage
+
+    val isValidSchool: LiveData<Boolean> = _school.map { school -> checkValidSchool(school) }
+
+    val isEmpty_department: LiveData<Boolean> =
+        _department.map { department -> checkEmpty_department(department) }
+    val isEmpty_studentid: LiveData<Boolean> =
+        _studentid.map { studentid -> checkEmpty_studentid(studentid) }
+    val isEmpty_name: LiveData<Boolean> =
+        _name.map { name -> checkEmpty_name(name) }
+    val isEmpty_id: LiveData<Boolean> =
+        _id.map { id -> checkEmpty_id(id) }
+    val isEmpty_code: LiveData<Boolean> =
```

Comment:
```suggestion
    val isEmptyDepartment: LiveData<Boolean> =
        _department.map { department -> checkEmptyDepartment(department) }
    val isEmptyStudentid: LiveData<Boolean> =
        _studentid.map { studentid -> checkEmptyStudentid(studentid) }
    val isEmptyName: LiveData<Boolean> =
        _name.map { name -> checkEmptyName(name) }
    val isEmptyId: LiveData<Boolean> =
        _id.map { id -> checkEmptyId(id) }
    val isEmptyCode: LiveData<Boolean> =
```
[안드로이드 공식문서에서 변수명과 함수명은 camelCase로 작성하도록 안내](https://developer.android.com/kotlin/style-guide#non-constant_names)하고 있습니다! 저희 [네이밍 컨벤션](https://www.notion.so/yell0/variable-3de143c1219942499ffd0e599fc9c383?pvs=4)도 한번씩 확인해서 수정 부탁드려요 🙃

Reply:
앗 바로 수정하겠습니다 !!